pom.xml

```diff
@@ -1434,4 +1434,4 @@
             </properties>
         </profile>
     </profiles>
-</project>
+</project>
```

Comment:
Don't change this.

Reply:
Sorry, it's my fault. I didn't realize that.